packages/api-client/src/components/SideNav/DownloadAppButton.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
+import SideNavLink from '@/components/SideNav/SideNavLink.vue'
+import { useLayout } from '@/hooks'
+import { ScalarIcon } from '@scalar/components'
+
+const { layout } = useLayout()
+</script>
+<template>
+  <SideNavLink
+    class="download-app-button sm:px-3"
```

Comment:
i think we should either decrease the padding in sections or increase it here to maintain alignment

Reply:
In sections?